modules/apps/27-interchain-accounts/controller/ibc_middleware_test.go

```diff
@@ -623,6 +663,10 @@ func (suite *InterchainAccountsTestSuite) TestOnTimeoutPacket() {
 			cbs, ok := suite.chainA.App.GetIBCKeeper().Router.GetRoute(module)
 			suite.Require().True(ok)
 
+			if isNilApp {
```

Comment:
Could you explicitly set to false? I believe if a test case is appended, it'll still be set to true

Reply:
nice catch, missed this on a few of the test functions